build.proj

```diff
@@ -56,7 +60,13 @@
     <MSBuild Projects="@(_ProjectsToRestore)" Targets="Restore" />
   </Target>
 
-  <Target Name="Test" />
+  <Target Name="Test">
+    <MSBuild Projects="@(UnitTestProjects)" Targets="VSTest"
+        ContinueOnError="ErrorAndContinue"
+        Properties="Verbosity=normal" />
```

Comment:
Let's just check to see if this is the best way to print out failures from running the tests. In general we try to avoid passing down global properties as they leak down into the the projects and mess up incremental build. cc: @ericstj

Reply:
Also if you're running tests as part of the build, you want to publish the xunit results in the yaml file as part of the build in CI so that test results show in the test explorer: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/publish-test-results?view=azure-devops&tabs=yaml#yaml-snippet